frontend/src/pages/instance/components/InstanceForm.vue

```diff
@@ -204,11 +205,41 @@
                         v-model="copyParts"
                     />
                 </template>
+                <div v-else-if="creatingNew && flowBlueprintsEnabled && flowBlueprints.length > 1">
```

Comment:
```suggestion
                <div v-else-if="creatingNew && flowBlueprintsEnabled && flowBlueprints.length > 0">
```

Think this was an off-by-one error. I only had one legit blueprint in my database, so nothing was being shown.